utils/node_local_test.py

```diff
@@ -3988,6 +3988,77 @@ def test_pydaos_kv(server, conf):
     daos._cleanup()
     log_test(conf, pydaos_log_file.name)
 
+
+def test_pydaos_kv_obj_class(server, conf):
+    """Test the predefined object class works with KV"""
+    # pylint: disable=consider-using-with
+
+    pydaos_log_file = tempfile.NamedTemporaryFile(prefix='kv_objclass_pydaos_',
+                                                  suffix='.log',
+                                                  delete=False)
+
+    os.environ['D_LOG_FILE'] = pydaos_log_file.name
+    daos = import_daos(server, conf)
+
+    pool = server.get_test_pool_obj()
+
+    cont = create_cont(conf, pool, ctype="PYTHON")
+
+    container = daos.DCont(pool.uuid, cont.uuid)
```

Comment:
I think pool.label or pool.id() would get you a label here, but I'm not sure if daos.DCont() accepts labels.
```suggestion
    container = daos.DCont(pool.label, cont.uuid)
```